tool/debian/build.sh

```diff
@@ -86,26 +86,22 @@ if [ -n "$BUILD_ID" ]; then
 fi
 
 FOLDER=$FOLDER_PREFIX-cpp
-if [ $CUDA = "ON" ]; then
-  FOLDER=$FOLDER_PREFIX-cuda$CUDA_VERSION-cudnn$CUDNN_VERSION
-fi
-
 if [ $PYTHON = "ON" ]
 then
   FOLDER=$FOLDER/python-singa
```

Comment:
the folder name decides the deb file name.
we'd better make them consistent. otherwise it would be confused if we later add a python-singa-cuda8.0-cudnn5 (the current singa-cuda8.0-cudnn5 does include the python part).
one solution is not differentiating python-singa and singa; 
another solution is differentiating them clearly (I prefer this).